backend/functions/setupProject.js

```diff
@@ -1,157 +0,0 @@
-const admin = require('firebase-admin');
```

Comment:
Maybe don't actually delete this but just rename it to something else or put it in a folder where it won't compile.